### ISSUE TITLE
Where is the list_file of CASIA-WebFace-112X96 dataset

### ISSUE THREAD

**Issue by Paleve** [yule-li/CosFace#10]
I download the preprocessed dataset of CASIA-WebFace-112X96, however I didn't get the list_file for training. Could you provide me the it, or give some methods?

**Comment by chenyyx** [comment]
I recreated a list of data myself. Put it here --------- Links : https://pan.baidu.com/s/1Vq-hwmEDI4eULYH6TmiApg password: buyc

**Comment by Paleve** [comment]
> I recreated a list of data myself. Put it here --------- Links : https://pan.baidu.com/s/1Vq-hwmEDI4eULYH6TmiApg password: buyc

非常感谢，想知道你使用CASIA-WebFace-112X96训练得到的结果怎么样，我训练60000 epoch得到的精度只有0.959，嵌入纬度1024，m设置为0.35，参数设置为：
  4 pretrained_model: None
  5 loss_type: cosface
  6 network: sphere_network
  7 data_dir: dataset/CASIA-maxpy-clean
  8 list_file: dataset/lst
  9 model_def: models.inception_resnet_v1
 10 max_nrof_epochs: 100
 11 batch_size: 90
 12 image_size: 160
 13 image_src_size: 256
 14 image_height: 112
 15 image_width: 96
 16 people_per_batch: 30
 17 num_gpus: 1
 18 images_per_person: 5
 19 epoch_size: 600
 20 alpha: 0.35
 21 scale: 64.0
 22 weight: 2.0
 23 embedding_size: 1024
 24 random_crop: False
 25 random_flip: True
 26 fc_bn: False
 27 keep_probability: 1.0
 28 weight_decay: 0.0001
 29 optimizer: MOM
 30 center_loss_factor: 0.0
 31 center_loss_alfa: 0.95
 32 learning_rate: -1.0
 33 learning_rate_decay_epochs: 100
 34 learning_rate_decay_factor: 1.0
 35 moving_average_decay: 0.9999
 36 seed: 666
 37 learning_rate_schedule_file: lr_coco.txt

**Comment by Paleve** [comment]
@chenyyx 难道使用112x96的图最好设置嵌入纬度512吗？

**Comment by chenyyx** [comment]
@Paleve 你好哈，我训练的 embedding_size 为 128，1024，唯独 512 的没有训练过，23333。128 的好一些，忘记 epoch 大小了，但是 1080ti 训练了 3 天左右，得到的精度是 99.3%，还不错。如果你实在没有提升的地方了，可以换换图片呀，换换 image_size ，我们这儿有人就是这么做的。效果也挺好的。

**Comment by Paleve** [comment]
@chenyyx 我用1080ti几个小时就跑了60000步，你这么说可能是我训的模型没完全收敛。非常感谢，这几天刚要入门人脸识别，方便留个邮箱吗有些问题希望能请教一下
